src/main/java/racing/Racing.java

```diff
@@ -25,4 +35,24 @@ private void updateCar() {
             participant.showDistance();
         }
     }
+
+    public void winner() {
+        int longest_distance = findLongestDistance();
+        String winner = "";
```

Comment:
```suggestion
        List<String> winners = new ArrayList<>();
```

우승자는 1명 이상일 수 있습니다.
하나의 문자열 데이터로 다루기보다는 우승자 목록으로 다루는게 더 바람직합니다.
또한 winner 라는 문자열형 변수로는 우승자가 1명인 것으로 유추할 수도 있습니다.
문자열에 계속 값을 더하는 방법보다는 눈에 더 확 띄는 목록형 데이터로 바꿔보세요.

객체지향은 코드 유지보수에도 큰 도움이 되지만 눈에 확 띄는 가시성 역시 좋아집니다.